src/cirrus/build.py

```diff
@@ -11,21 +11,43 @@
 """
 import os
 import sys
+from argparse import ArgumentParser
 
 from cirrus.environment import cirrus_home
 from cirrus.configuration import load_configuration, get_pypi_auth
 from fabric.operations import local
 
 
-def main():
+def build_parser(argslist):
     """
-    _main_
+    _build_parser_
+
+    Set up command line parser for the build command
+
+    """
+    parser = ArgumentParser(
+        description = 'git cirrus build'
+    )
+    parser.add_argument('command',nargs='?')
+    parser.add_argument(
+        '-c',
+        '--clean',
+        action='store_true',
+        help='remove existing virtual environment')
+    opts = parser.parse_args(argslist)
+    return opts
+
+
+def execute_build(opts):
+    """
+    _execute_build_
 
     Execute the build in the current package context.
 
     - reads the config to check for custom build parameters
       - defaults to ./venv for virtualenv
       - defaults to ./requirements.txt for reqs
+    - removes existing virtualenv if clean flag is set
     - builds the virtualenv
     - pip installs the requirements into it
```

Comment:
Could you document `opts` and its type? We typically use sphinx style `:param FooType opts: A FooType of blah`